src/routes/components/index.svelte

```diff
@@ -3,6 +3,7 @@
   import List from "$lib/components/ComponentIndex/CardList.svelte";
   import Button from "$lib/components/ComponentIndex/ArrowButton.svelte";
   import components from "./components.json";
+	import { compare } from '$lib/utils/sort';
```

Comment:
it looks like this file is indented using spaces, but this line is a tab

Reply:
thx, done ✅